training/cogvideox_image_to_video_lora.py

```diff
@@ -201,6 +200,64 @@ def log_validation(
     return videos
 
 
+def run_validation(
```

Comment:
Didn't we already have a utility to do this? 
https://github.com/a-r-r-o-w/cogvideox-factory/blob/bd4062760c75bad910e65bd1da107cf7f399dc30/training/cogvideox_image_to_video_lora.py#L153